packages/marshal/src/marshal.js

```diff
@@ -480,7 +480,18 @@ const defaultSlotToValFn = (x, _) => x;
 export function makeMarshal(
   convertValToSlot = defaultValToSlotFn,
   convertSlotToVal = defaultSlotToValFn,
+  // Messages are the medium
+  { marshalName = 'McLuhan' } = {},
```

Comment:
Is this a reference to this? https://www.youtube.com/watch?v=ImaH51F4HBw

Reply:
Thanks for reminding me of that. I didn't know about the YouTube vid but altogether yes. Marshal McLuhan was a famous media theorist, with "The Medium is the Message" being his famous tag line.

Although I amused myself for a moment with this obscure cultural reference, when I needed a default marshalName, it does not belong in production code. I just changed the default to `'anon-marshal'`.

Reply:
sounds good!

Reply:
Aside:

My default Marshall is Thurgood.

Etymology of marshal came up the other day. The mar is mare (equestrian) and shal is farrier (as in iron, smith). Something to do with being in charge of lining up horses.